examples/keras_track_distribution.py

```diff
@@ -0,0 +1,59 @@
+import aim
+from aim import track
+
+import numpy as np
+import mnist
+import keras
+from keras.models import Sequential
+from keras.layers import Dense
+from keras.utils import to_categorical
+
+"""
+Citation for Keras Example Model: victorzhou.com/blog/keras-neural-network-tutorial/
+"""
+
+class TrackCallBack(keras.callbacks.Callback):
+  def on_epoch_end(self, epoch, logs=None):
```

Comment:
- It would be better to not only track weights on epoch end, but also on iteration end and on some condition(e.g. on each 2*k iteration).
 - Also it would be good to move `TrackCallBack` into SDK and make it importable, so users can just import and use it.

Reply:
Hi- Now, I track weights on epoch end and every k-th iteration end. I have also moved TrackCallBack in track.py in SDK so that it is importable.